requirements.txt

```diff
@@ -49,7 +49,8 @@ pathlib2==2.3.0
 polyline==1.3.2
 py==1.7.0                 # via retry
 pycparser==2.19           # via cffi
-pydruid==0.5.0
+#pydruid==0.5.0
+git+https://github.com/switchdin/pydruid.git@registered-lookups # pydruid
```

Comment:
We can't do this as it's a security issue. Get your feature merged / released on `pydruid` first.

Reply:
okay, fixed! thx

Reply:
@srggrs still no release yet?